deployment/Deploy.ps1

```diff
@@ -236,13 +236,17 @@ process {
             }
         }
     
-        #Updating the Config Entry EdnaLiteDevKey in the Function Config
-        $KeyVaultLink=$(az keyvault key show --vault-name $deploymentOutput.properties.outputs.KeyVaultName.value --name EdnaLiteDevKey --query 'key.kid' -o json);
+        #Creating EdnaLiteDevKey in keyVault and Updating the Config Entry EdnaLiteDevKey in the Function Config
+        $ke = az keyvault key create --vault-name $deploymentOutput.properties.outputs.KeyVaultName.value --name EdnaLiteDevKey --protection software
```

Comment:
If this fails, should we even proceed with the script 🤔 ?

Reply:
updating this in the followup